checkpoints/ten_readme.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/README.md b/README.md
```

Comment:
Since this is a patch, can you add FE tasks as well?
* Handle Tweet component for liking / un-liking the tweet
* Handle storing of the like and un-like in the backend